백다연/MvvmStudy/app/src/main/java/com/test/mvvmstudy/di/ApiModule.kt

```diff
@@ -0,0 +1,52 @@
+package com.test.mvvmstudy.di
+
+import com.test.mvvmstudy.BuildConfig
+import com.test.mvvmstudy.api.GithubApi
+import com.test.mvvmstudy.util.GITHUB_DOMAIN
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object ApiModule {
+
+    @Provides
+    fun provideBaseUrl() = GITHUB_DOMAIN
+
+    @Provides
+    fun getLogging() = HttpLoggingInterceptor().apply {
+        level = if (BuildConfig.DEBUG) {
+            HttpLoggingInterceptor.Level.BODY
+        } else HttpLoggingInterceptor.Level.NONE
+    }
+
+    @Provides
+    fun getClient() = OkHttpClient.Builder()
+        .addNetworkInterceptor(getLogging())
+        .build()
+
```

Comment:
provide로 통일하면 더 좋을 것 같아요!
```suggestion
    fun provideHttpClient() = OkHttpClient.Builder()
        .addNetworkInterceptor(getLogging())
        .build()

```